src/main/kotlin/frc/robot/subsystems/IntakeSubsystem.kt

```diff
@@ -0,0 +1,108 @@
+package frc.robot.subsystems
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.SparkMaxAbsoluteEncoder
+import com.revrobotics.CANSparkMaxLowLevel
+import edu.wpi.first.math.controller.PIDController
+import edu.wpi.first.math.controller.ArmFeedforward
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.wpilibj.Encoder
+import edu.wpi.first.wpilibj.motorcontrol.MotorController
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+
+class IntakeSubsystem(intakeMotorLID: Int,
+                      intakeMotorRID: Int,
+                      pitchMotorID: Int,
+) : SubsystemBase() {
+
+    val intakeMotorR = CANSparkMax(intakeMotorRID, CANSparkMaxLowLevel.MotorType.kBrushless)
+    val intakeMotorL = CANSparkMax(intakeMotorLID, CANSparkMaxLowLevel.MotorType.kBrushless)
+    val pitchMotor = CANSparkMax(pitchMotorID, CANSparkMaxLowLevel.MotorType.kBrushless)
+    var intakeMotors = MotorControllerGroup(intakeMotorL, intakeMotorR)
+
+    val pitchEncoder = pitchMotor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle)
+    //var elbowPos = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowPositions")
+    // var desiredIntakePitch = NetworkTableInstance.getDefault().getTable("Intake").getDoubleTopic("IntakePositions").publish()
+    var desiredIntakeSpeed = NetworkTableInstance.getDefault().getTable("Intake").getDoubleTopic("IntakePositions").publish()
+    var desiredIntakePitch = NetworkTableInstance.getDefault().getTable("Intake").getDoubleTopic("IntakePositions").publish()
+    var intakePosition = NetworkTableInstance.getDefault().getTable("Intake").getDoubleTopic("IntakePositions").publish()
+
+
+
+    val pitchFFS = 0.0
+    val pitchFFG = 0.0
+    val pitchFFV = 0.0
+    val pitchFFA = 0.0
+    val FFVelocity = 0.0
+
+    var intakeFeedforward = ArmFeedforward(pitchFFS,pitchFFG, pitchFFV, pitchFFA)
+
+    val pitchP = 0.0
+    val pitchI = 0.0
+    val pitchD = 0.0
+
+    val intakePID = PIDController(pitchP,pitchI,pitchD)
+
+    val pitchEncoderPositionConversionFactor = 0.0
+    val pitchEncoderVelocityConversionFactor = 0.0
```

Comment:
there is some code i can find for you to get the proper conversion factors, i will give it to you